console/console-init/ui/src/modules/msg-and-iot/dailogs/components/IoTReviewStep/IoTReviewStep.tsx

```diff
@@ -4,14 +4,11 @@
  */
 
 import React from "react";
-import {
-  isMessagingProjectValid,
-  isIoTProjectValid
-} from "modules/msg-and-iot/dailogs/utils";
-import { IIoTProjectInput } from "modules/msg-and-iot/dailogs/components";
+import { isIoTProjectValid } from "modules/msg-and-iot/dailogs";
+import { IIoTProjectInput } from "modules/msg-and-iot/dailogs";
 import { IoTReview } from "../IoTReview";
 
-const IoTProjectReview = (projectDetail?: IIoTProjectInput) => {
+const IoTReviewStep = (projectDetail?: IIoTProjectInput) => {
   const isReviewEnabled = () => {
```

Comment:
can we try variable name as isEnabledReviewButton, i think isEnabled prefix have more sense.